examples/bigquery-analyze-realtime-reddit-data/terraform/main.tf

```diff
@@ -0,0 +1,284 @@
+# service account for reddit VM
+resource "google_service_account" "reddit_service_account" {
+  account_id   = "${var.service_account_name}"
+  display_name = "Reddit VM Service Account"
+}
+
+resource "google_project_iam_binding" "reddit_vm_compute" {
+  project = "${var.project_id}"
+  role    = "roles/compute.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+resource "google_project_iam_binding" "reddit_vm_dataflow" {
+  project = "${var.project_id}"
+  role    = "roles/dataflow.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+resource "google_project_iam_binding" "reddit_vm_pubsub" {
+  project = "${var.project_id}"
+  role    = "roles/pubsub.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+
+resource "google_compute_instance" "reddit_vm" {
+  name         = "reddit-data-collector-1"
+  machine_type = "e2-micro"
+  zone         = "us-central1-a"
+
+
+  boot_disk {
+    initialize_params {
+      image = "centos-cloud/centos-7"
+    }
+  }
+
+  network_interface {
+    network = "default"
+
+    access_config {
+      // Ephemeral public IP
+    }
+  }
+
+  metadata_startup_script = <<EOF
+    echo \"install python and git...\"
+    yum install python3 -y
+    yum install git -y
+    echo \"installing pip...\"
+    curl https://bootstrap.pypa.io/pip/3.6/get-pip.py -o get-pip.py
+    python3 get-pip.py
+    python3 -m pip install --upgrade pip
+    echo \"importing python libraries...\"
+    python3 -m pip install praw
```

Comment:
How about a pip install -r Requirements.txt with the other repo you are cloning further down?